start.js

```diff
@@ -246,7 +253,7 @@ const haveEnoughResources = ({ fleet, shipInfo }, nowSec) => {
     const max = shipInfo[`${resource}MaxReserve`] * shipAmount;
     const current = left * shipAmount;
     const needed = max - current;
-    if (resourceAvailability[resource] < needed) {
+    if (inventory[resource] < needed) {
```

Comment:
@MartinGerritsen This does not work and is the source of my issue #9 
`inventory[resource]` where resource = "arms" -> inventory["arms"] is undefined
it should be like:
`inventory[2]["amount"]`
to get the value needed for this condition.
It does not get to the point to set `enoughResources = false`
As soon as I forced the if condition to be true the program did buy the resources.

Reply:
>If you want to get rid of resourceAvailability and keep using inventory (which seems to be a better solution than mine)
>Then change line 247-249 in start.js
>```
>if (resourceAvailability[resource] < needed) {
>      enoughResources = false;
>}
>```
>as of your last commit this will never get false because resourceAvailability is undefined. You got to get the data from inventory. I am not that firm in javascript. So do not count on me :D

_Originally posted by @speedyconzales in https://github.com/Heimdall-Industries/Heidrun/pull/4#discussion_r995660861_